### PR TITLE
tests left

### PR DESCRIPTION
**Pull request by jomivega**

### CODE REVIEW

**Comment by jomivega** [comment]
Thats a first! I passed travis on the first commit 😃😃

File: tests/test_trading_model.py [hunk 1]
```diff
@@ -31,7 +32,7 @@ def test_trading_acc_av_stk():
     """
     Test available stocks
     """
-    user = User.objects.create(username='christophe', password="iscool")
+    user = create_user()
```
Comment (Neitsch) [inline tests/test_trading_model.py]: Good idea! To optimize this, please use a fixture:
```
@pytest.fixture
def user():
    pwd = ''.join(random.choices(string.ascii_uppercase + string.digits, k=9))
     user = User.objects.create(username='user', password=pwd)
     return user

def test_trading_acc_av_stk(user):
```

File: tests/test_trading_model.py [hunk 2]
```diff
@@ -183,10 +184,7 @@ def test_trading_trading_balance():
     """
     Testing available_cash for a Trading Account
     """
-    pwd = ''.join(random.choices(string.ascii_uppercase + string.digits, k=9))
-    user = User.objects.create(username='user', password=pwd)
-    account = TradingAccount(profile=user.profile, account_name="testAccount")
-    account.save()
+    account = create_account()
```
Comment (Neitsch) [inline tests/test_trading_model.py]: Here you can also use a fixture

File: tests/test_trading_model.py [hunk 3]
```diff
@@ -216,3 +214,152 @@ def test_trading_trading_balance():
...
+        name="Name1X",
+        ticker="TKRC"
+    )
+    stock1.daily_quote.create(
+        value=value_of_stock1,
+        date="2016-06-03"
+    )
+
+    current_value = TradeStock(quantity=1, account=account, stock=stock1).current_value()
+    assert current_value == -value_of_stock1
```
Comment (Neitsch) [inline tests/test_trading_model.py]: Preferrrable quantity != 1 to verify that the quantity is considered.

File: tests/test_trading_model.py [hunk 4]
```diff
@@ -216,3 +214,152 @@ def test_trading_trading_balance():
...
+
+@pytest.mark.django_db(transaction=True)
+def test_tradebucket_current_value():
+    """
+    Tests TradeBucket.current_value()
+    """
+    pwd = ''.join(random.choices(string.ascii_uppercase + string.digits, k=9))
+    user = User.objects.create(username='user', password=pwd)
+    account = TradingAccount(profile=user.profile, account_name="testAccount")
+    account.save()
```
Comment (Neitsch) [inline tests/test_trading_model.py]: Use a fixture

File: tests/test_stocks_model.py [hunk 5]
```diff
@@ -321,3 +321,32 @@ def test_bucket_change_config():
...
+        date="2016-06-06"
+    )
+    bucket = InvestmentBucket(name="bucket", public=True, owner=user.profile, available=2)
+    bucket.save()
+    config = InvestmentStockConfiguration(
+        quantity=1,
+        stock=stock,
+        bucket=bucket,
+        start="2016-06-08"
+        )
```
Comment (Neitsch) [inline tests/test_stocks_model.py]: Same here, quantity != 1 is preferred.

**Review by Neitsch — COMMENTED** [review]
Overall, really good tests. Just small suggestions :)

**Review by Neitsch — CHANGES_REQUESTED** [review]
plz address comments

**Review by Neitsch — APPROVED** [review]

**Pull request merged**